dips/dip-196.md

```diff
@@ -0,0 +1,55 @@
+---
+dip: 196
+title: Onchain Network Identity
+authors: Greg Nazario (@gregnazario)
+status: Draft
+type: Informational
+created: 07/06/2021
+last_updated: 07/06/2021
+issue: https://github.com/diem/dip/issues/196
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Summary
+
+Diem nodes need the ability to find other nodes in the network easily.  Additionally, the nodes need the ability to find 
+the associated network identity so that the nodes can trust them.  This DIP describes the ability for node owners to
```

Comment:
Is node owner a formal term?  E.g. in DIP-6 Validator Owner?  If so we should still to the terminology or if generic, perhaps node operator might be a more appropriate.

Reply:
Yeah, can change to `node operator`, the differences between `owner` and `operator` are blurred then at the full node usage.